### PR TITLE
Release notes for 1.0

### PR DESCRIPTION
**Pull request by ctb**
This updates the screed description & closes https://github.com/dib-lab/khmer/issues/1118.

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make clean diff-cover` If it introduces new functionality, is it tested?
- [x] `make format diff_pylint_report doc` Is it well formatted?
- [x] Is it documented in the `ChangeLog`?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by ctb** [comment]
Ready for review & merge @standage @luizirber

File: doc/release-notes/RELEASE-1.0.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Release 1.0
+
+We are pleased to announce the release of Screed v0.9. Screed is a database
+engine capable of storing and retrieving short-read sequence data and is
+designed to be fast and adaptable to different sequence file formats.
+
+We are pleased to announce the release of screed 1.0. Screed is a
+biological sequence parsing and storage/retrieval library for DNA and
+protein sequences. It's designed to be lightweight and easy to use
+from Python.
```
Comment (standage) [inline doc/release-notes/RELEASE-1.0.md]: This paragraph has 1.0, but has older(?) language.

File: doc/release-notes/RELEASE-1.0.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Release 1.0
+
+We are pleased to announce the release of Screed v0.9. Screed is a database
+engine capable of storing and retrieving short-read sequence data and is
+designed to be fast and adaptable to different sequence file formats.
```
Comment (standage) [inline doc/release-notes/RELEASE-1.0.md]: This paragraph is marked 0.9 but has better (and newer?) language. Duplicated paragraph below?

File: doc/release-notes/RELEASE-1.0.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## New items of note:
+
+## Known Issues
+
+These are all pre-existing
+
+ - Screed does not support gzip file streaming. This is an issue with Python 2.x and will likely *not* be fixed in future releases. This is being tracked in ged-lab/khmer#700
+ 
+ - Screed is overly tolerant of spaces in fast{a,q} which is against spec. This is being tracked in ged-lab/khmer#108
```
Comment (standage) [inline doc/release-notes/RELEASE-1.0.md]: s/ged-lab/dib-lab/

File: doc/release-notes/RELEASE-1.0.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ 
+## Contributors
+
+@luizirber @mr-c @bocajnotnef @ctb \*@proteasome \*@anotherthomas \*@sguermond
```
Comment (standage) [inline doc/release-notes/RELEASE-1.0.md]: Needs updating.

File: doc/release-notes/RELEASE-1.0.md [hunk 5]
```diff
@@ -1,9 +1,5 @@
 # Release 1.0
 
-We are pleased to announce the release of Screed v0.9. Screed is a database
-engine capable of storing and retrieving short-read sequence data and is
-designed to be fast and adaptable to different sequence file formats.
-
```
Comment (standage) [inline doc/release-notes/RELEASE-1.0.md]: Lol, I like this paragraph better.

**Comment by ctb** [comment]
but it's less accurate...

**Comment by standage** [comment]
Still need:

- [ ] author list update
- [ ] fill out **New items of note**

Unless directed otherwise, I'll leave this PR open for those two points.

**Comment by luizirber** [comment]
I updated `setup.py` to include all the authors, and also added a clearer description (the same from the release notes).

Ready for review and merge @standage @betatim @ctb

**Pull request merged**